src/utils/frontend-api-client/work-orders.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios'
```

Comment:
I think this file has been superseded by src/utils/frontend-api-client/repairs.js now and can be deleted.